### DISCUSSION TITLE
beonlabs community script for Web Follower is not available

### DISCUSSION THREAD

**Discussion by tqkam** [NightscoutFoundation/xDrip#3125] [Q&A]
Hi,
the source of the custom script https://community-script.beonlabs.net returns a file with text "service ceased". So that xDrip doesn't get data from Libre 3 anymore.
How could the problem be solved?

**Comment by Navid200** [comment]
Have you followed these instructions?
https://navid200.github.io/xDrip/docs/Libre3_Follow.html

**Comment by tqkam** [comment]
Hi,
1) I followed the instructions.
2) I view the code to find a source of the problem.
Here it is: https://github.com/NightscoutFoundation/xDrip/blob/master/app/src/main/java/com/eveningoutpost/dexdrip/cgm/webfollow/Template.java 

`private static String getUrl() {
        val ck = Cpref.get("CK");
        if (JoH.emptyString(ck)) return null;
        if (!ck.contains(".")) {
            return SCHEME + PREFIX + DOT + ck + DOT + SUFFIX;
        }
        return ck;
    }`
The URL returned is https://community-script.beonlabs.net.

3) The issue I see is, this script, which contains commands to get the Data from LibreLinkUp API, it doesn't exist anymore.

I'm sure I'm not so wrong with the instructions because I've used the LibreLinkUp already, and I've also played with LibreLinkUp API to get   sugar values.

**Comment by Raimund58** [comment]
That explains why my xDrip suddenly stopped getting data if the script silently went missing.

**Comment by Raimund58** [comment]
The script is still empty if I download it, but I get data again through the web follower.
